Best practices/React/Testing.md

```diff
@@ -163,3 +163,46 @@ Another benefit of testing React components is that they typically abstract away
     });
   });
   ```
+
+* If you find that you need to share utilities between test files (for example, because you have split a single component’s tests among several files), put them in the nearest shared `tests/utilities` directory. This differentiated test files from "support" files, and mirrors how we store fixtures.
+
+  ```
+  # Bad
+  MyComponent/
+  └── tests
+      ├── feature-one.test.tsx
+      ├── feature-two.test.tsx
+      └── utilities.ts
+
+  # Good
+  MyComponent/
+  └── tests
+      ├── feature-one.test.tsx
+      ├── feature-two.test.tsx
+      └── utilities/
+          └── index.ts
+  ```
+
+### Split test files
+
+As noted in the [decision record](../../Decision%20records/06%20-%20We%20split%20up%20large%20component%20test%20files%20by%20feature.md), when a component’s test file gets large enough that is hard to navigate and/ or stresses editor tooling, we split the test file by feature. Below are some best practices for naming and structure of these split files:
+
+* Name the split files in the format `component-feature.test.tsx`. For example, a test file for the SEO feature of a `ProductDetails` component would be found at `ProductDetails/tests/seo.test.tsx`.
```

Comment:
this might want to be `ProductDetail/tests/seo-card.test.tsx` or `ProductDetail/tests/SEOCard.test.tsx`

Reply:
I considered just naming it the component but felt it was wonky for two reasons:

1. Some tests around SEO will not be about the SEO card. For instance, we may need to normalize some values that just happens when we do a mutation
2. It creates a lot of duplicate file names (which is the same problem as my existing naming convention, I like @asmockler's idea above)